### ISSUE TITLE
Add small hyperparameter tuning ( keep the default values in the grid to avoid any accuracy loss ! )

### ISSUE THREAD

**Issue by Brice-Vergnou** [Brice-Vergnou/create-your-spotify-recomendation#4]
https://github.com/Brice-Vergnou/create-your-spotify-recomendation/blob/576355cbc71198b15f3f3c44dcb12f08b68d21d7/get_your_model.py#L129-L129

**Comment by Brice-Vergnou** [comment]
Successfully did thanks to pycaret (it's not hyper-parameter tuning, but it finds the best model ).
